packages/kolibri-tools/lib/i18n/utils.py

```diff
@@ -6,22 +6,74 @@
 import os
 import sys
 
-import kolibri_exercise_perseus_plugin
+try:
+    import kolibri_exercise_perseus_plugin
+
+    PERSEUS_LOCALE_PATH = os.path.join(
+        os.path.dirname(kolibri_exercise_perseus_plugin.__file__), "locale"
+    )
+    PERSEUS_SOURCE_PATH = os.path.join(PERSEUS_LOCALE_PATH, "en", "LC_MESSAGES")
+    if not os.path.exists(PERSEUS_SOURCE_PATH):
+        os.makedirs(PERSEUS_SOURCE_PATH)
+    if not os.path.exists(PERSEUS_LOCALE_PATH):
+        os.makedirs(PERSEUS_LOCALE_PATH)
+except:
+    PERSEUS_LOCALE_PATH = None
+    PERSEUS_SOURCE_PATH = None
+    pass
```

Comment:
a lot of this complexity would go away with a vendored perseus @rtibbles

Reply:
I'll get back to you on that...